pkg/sidecar/config/config.go

```diff
@@ -229,38 +221,22 @@ func (s *ScyllaConfig) setupEntrypoint(ctx context.Context) (*exec.Cmd, error) {
 		}
 	}
 
-	// See if we need to use cpu-pinning
-	// TODO: Add more checks to make sure this is valid.
-	// eg. parse the cpuset and check the number of cpus is the same as cpu limits
-	// Now we rely completely on the user to have the cpu policy correctly
-	// configured in the kubelet, otherwise scylla will crash.
-	if cluster.Spec.CpuSet {
-		cpusAllowed, err := getCPUsAllowedList("/proc/1/status")
-		if err != nil {
-			return nil, errors.WithStack(err)
-		}
-		if err := s.validateCpuSet(ctx, cpusAllowed, s.cpuCount); err != nil {
-			return nil, errors.WithStack(err)
-		}
-		args["cpuset"] = &cpusAllowed
+	cpusAllowed, err := getCPUsAllowedList("/proc/1/status")
+	if err != nil {
+		return nil, errors.WithStack(err)
 	}
+	if err := s.validateCpuSet(ctx, cpusAllowed, s.cpuCount); err != nil {
+		return nil, errors.WithStack(err)
+	}
+	args["cpuset"] = &cpusAllowed
 
-	version := semver.NewScyllaVersion(cluster.Spec.Version)
-
-	klog.InfoS("Scylla version detected", "version", version)
-
-	if len(cluster.Spec.ScyllaArgs) > 0 {
-		if !version.SupportFeatureUnsafe(semver.ScyllaVersionThatSupportsArgs) {
```

Comment:
should this also be removed from validation then?

Reply:
done